etl/steps/data/garden/papers/2023-12-12/nemet_2009.meta.yml

```diff
@@ -1,2 +1,24 @@
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Energy
+        - Technological Change
+
 dataset:
-  title: Cost and capacity of photovoltaic technology (Nemet, 2009)
+  title: Cost and capacity of photovoltaic technology
+  update_period_days: 0
+
+tables:
+  nemet_2009:
+    variables:
+      cost:
+        title: Cost of photovoltaic technology
+        units: 2004 USD/Watt
```

Comment:
Should be `unit`, not `units`.

Reply:
Ouch!! In my new laptop I had not yet properly configured the path to the yaml definitions schema. Now I see it as an issue. Thanks for pointing it out, fixed!